### ISSUE TITLE
Game messed up on start

### ISSUE THREAD

**Issue by Kusarikage** [MelvorIdle/melvoridle.github.io#638]
**Describe the bug**
I wanted to start MelvorIdle with steam a few minutes ago and got to the Choose-Cloud-Save Screen. I clicked on the appropiate Savegame as always but then the whole Overlay got messed up like you can hopefully see in the (hopefully) provided picture.
If there is no picture, I´ll explain quickly: The background has the standard, violet/blue mountain image, in the top left is the symbol for not being able to load an image. At the top there is some text without any textbox or anything: "A game by Malcs", "Also aviable on [Symbols for other platforms]" and "Melvor Idle is currently in active Alpha Development.
Under that is a textbox with the Early Access "warning", with the 5 points we agree to accept. At the bottom of the text box is a huge button that does nothing.
Under the textbox is the headline "Create or Load a Character" and under that are two buttons, which do nothing. Fun Fact: I was cutting wood before I quit last time and now in the broken window there are logs appearing, so a safe seems to actually be loaded.
PS: The Scroll bar can be dragged but nothing is moving up or down.

**To Reproduce**
Two-Monitor Setup and this is what I did in detail.
1) Open Melvor Idle in first, smaller Monitor.
2) During startup realize it´s in the wrong monitor and drag it over.
3) Let the game start and click on a... local(?) save file.
2) Broken Window

**Expected behavior**
I expected the usual window. Left bar with the hobbies, Middle window with the opened hobby... the usual.

**Screenshots**
![Screenshot 12 01 2021 03_56_08 (2)](https://user-images.githubusercontent.com/61149286/104264059-34536500-548b-11eb-9bbb-a8597357d509.png)
I hope I´m doing this right...

**Browser**
Please tell us what browser you are using and what version.
I use the bought Steam Version and I don´t know where to see a version number but it´s the most current as there´s no MelvorIdle update in steam.

**Console output**
Please open the console by pressing F12 and copy it here.
If I have to do this in the game window then I sadly have to say: Nothing happens.

**Are you using any scripts?**
Please let us know if you are using scripts the time this bug occurred.
Nope, used as installed since I got it.

**Comment by Kusarikage** [comment]
Update: De- and reinstalled the game and I still get this when I start the game. 
What I forgot to say is that after the initial broken window, that appeared after choosing a save, every other start after throws me right into this window upon starting without giving the option to choose anything.

**Comment by Kusarikage** [comment]
Update: Seriously, I have no idea what I did. The definition of madness is to do the same thing again and again and in madness I was as I opened the steam version again and again and now, out of nowhere it functions again.
Instead of the picture posted above I get straight to the "Choose Profile" page as it´s normal and from there I can access the game normally again. I really wish I could help to understand this bug better but honestly I have no clue. My best guess has to be it has something to do with my dual monitor setup as the bug appeared when I dragged the game across monitors upon startup but it´s a guess at best. 
No idea if this description sounds neutral or bad so let me just say that I absolutely adore this game and am happy I can play it more easy again!
